ops/mainnet/prod/core/main.tf

```diff
@@ -435,7 +435,7 @@ module "ecs" {
   stage                   = var.stage
   environment             = var.environment
   domain                  = var.domain
-  ecs_cluster_name_prefix = "nxtp-ecs"
+  ecs_cluster_name_prefix = "ecs"
```

Comment:
would change the name here to be "amarok" or something more descriptive

Reply:
Don't want to use amarok either. Maybe "bridge-ecs"?

Reply:
What would bridge stand for?
But actually, this is just the prefix -- the cluster specification will be baked in into its name, so I think we're good with just ECS.

Reply:
OK will move back to ecs.